CONTRIBUTING.md

```diff
@@ -1,38 +1,75 @@
+If you are familiar with a [GitHub-based pull request contribution workflow](https://help.github.com/articles/about-pull-requests/), 
+please note that **most issues are subject to quite a bit of discussion** before 
+draft language becomes ready for a pull request. Please ensure the full problem 
+space of the issue is explored in the discussion, and that consensus has been reached 
+by the participating members, before contributing to the 
+repository.
 
-```
-+@github_username
-```
+## Normative and/or substantive contributions
 
-If you added a contributor by mistake, you can remove them in a comment with:
+The CSS Working Group operates via the consensus of its membership, and discusses 
+all significant matters prior to implementation.
 
-```
--@github_username
-```
+If adding an new, normative issue, please label/tag the PR as 'Agenda+' to bring 
+it to the Working Group's attention.
 
-If you are making a pull request on behalf of someone else but you had no part in designing the 
-feature, you can remove yourself with the above syntax.
+The Working Group, aside from managing issues on GitHub, mainly discusses specifications 
+and requests on [the www-style public mailing list](https://lists.w3.org/Archives/Public/www-style/), 
+and in CSSWG meetings.
 
-# Tests
+### After discussion
+
+Once the Working Group has come to a consensus, a contributor may file a pull request 
+against the issue. The editor of the specification related to the issue will then be 
+responsible for reviewing both the content and implementation of the contributed 
+changes.
+
+Please follow the [Pull Request template](https://github.com/w3c/csswg-drafts/blob/master/.github/PULL_REQUEST_TEMPLATE.md) 
+when contributing to the repository.
+
+### Tests
 
 For normative changes for any specification in
 [CR or later](https://www.w3.org/Style/CSS/current-work) as well as the pre-CR specifications listed
 below, a corresponding [web-platform-tests](https://github.com/w3c/web-platform-tests) PR must be
 provided, except if testing is not practical; for other specifications it is usually appreciated.
+
 Typically, both PRs will be merged at the same time. Note that a test change that contradicts the
 spec should not be merged before the corresponding spec change. If testing is not practical, please
 explain why and if appropriate [file an issue](https://github.com/w3c/web-platform-tests/issues/new)
 to follow up later. Add the `type:untestable` or `type:missing-coverage` label as appropriate.
 
 The pre-CR specifications with this testing requirement are currently:
 
-* cssom
-* cssom-view
+  * cssom
+  * cssom-view
+
+## Non-substantive contributions
+
+For simple spelling, grammar, or markup fixes unrelated to the substance of a specification, 
+please file an issue with the usual shortname tagging (ie. `[css-foo]`) for the spec(s) 
+edited by the change. You may then issue a pull request referencing the issue number.
```

Comment:
As I think I mentioned, simple fixes like this can just get a PR without an issue being filed (although if there is one, the PR should attempt to close it).